README.md

```diff
@@ -202,6 +206,22 @@ jobs:
           recreate: true 
 ```
 
+## Delete a comment which is no longer relevant
+
+Take a case where you need to delete a comment which is no longer relevant. E.g., we use this to write redirects for our Docs' site. If a contributor were to revert a filename change or a file deletion, we'd need to invalidate the comment. The `delete` flag will help in achieving this scenario.
+
+```yml
+- name: Delete comment if no redirects needed
+  if: steps.check-renamed.outputs.diff-status == 'false'
+  uses: hasura/comment-progress@v2.1.0
+  with:
+    github-token: ${{ secrets.GITHUB_TOKEN }}
+    repository: 'my-org/my-repo'
+    number: ${{ github.event.number }}
+    id: redirects-summary
+    delete: true
+```
```

Comment:
Maybe something more generic that leaves out our implementation specific stuff like `steps.check-renamed.outputs.diff-status`

```yml
on:
  pull_request:
    types: [closed]
    
jobs:
  cleanup-automated-comments:
    runs-on: ubuntu-20.04
    name: Delete automated PR comments
    steps:
      - name: delete comment that contains a preview link
        uses: hasura/comment-progress@v2.2.0
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          repository: 'my-org/my-repo'
          number: ${{ github.event.number }}
          id: preview-url
```